src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[],
+  creating: Todo | null,
+  onDelete: (todoId: number) => void,
+  inProcessing: number[],
+  updateTodo: (todoId: number [], data: boolean) => void,
+  onChangeTitle: (id: number, title: string) => void,
+};
+
+export const TodoList: React.FC<Props> = React.memo(
+  ({
+    todos,
+    creating,
+    onDelete,
+    inProcessing,
+    updateTodo,
+    onChangeTitle,
+  }) => {
+    return (
+      <section className="todoapp__main" data-cy="TodoList">
+        <TransitionGroup>
+          {todos.map(todo => {
+            return (
+              <CSSTransition
+                key={todo.id}
+                timeout={300}
+                classNames="item"
+              >
+                <TodoItem
+                  todo={todo}
+                  onDelete={onDelete}
+                  isProcessedIDs={inProcessing.includes(todo.id)}
+                  updateTodo={updateTodo}
+                  onChangeTitle={onChangeTitle}
+                />
+              </CSSTransition>
+            );
+          })}
+
+          {creating && (
+            <CSSTransition
+              key={0}
+              timeout={300}
+              classNames="temp-item"
+            >
+              <TodoItem
+                todo={creating}
+                onDelete={onDelete}
+                isProcessedIDs
```

Comment:
Add the prop value for `isProcessedIDs` (e.g. `isProcessedIDs={inProcessing.includes(creating.id)}`) or remove the prop if it's not needed.